src/main/resources/data/cakewars/lang/ru_ru.json

```diff
@@ -0,0 +1,56 @@
+{
+	"game.cakewars.beehive_cake_wars": "Cake Wars: Улей",
+	"game.cakewars.laboratory_cake_wars": "Cake Wars: Лаборатория",
+	"game.cakewars.meow_mountain_cake_wars": "Cake Wars: Мяу-Гора",
+	"game.cakewars.skylands_cake_wars": "Cake Wars: Небесные земли",
+	"gameType.cakewars.cake_wars": "Cake Wars",
```

Comment:
I think it would make more sense to translate the game name here like existing translations for other games.

Reply:
There is a problem that names like "Cake Wars", "Bed Wars", etc was transliterated or even not translated on russian servers since the beggining of this games. So now translated names of this games are very strange to russian players. And because of that I decided to not translate that name, but I think we can just transliterate it. What do you think?

Reply:
There are already some examples of Russian translations of game names, so I would just follow the existing cases:

- Codebreaker: Взломщик кода
- Creeperfall: Криперопад
- Gomoku: Гомоку
- Infinite Parkour: Бесконечный паркур
- Item Hunt: Охота за предметами
- Spleef: Сплиф
- Withersweeper: Визер-сапёр

Reply:
ok